app/views/content_items/publication_with_tasklist_sidebar.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: t("content_item.schema_name.#{@content_item.document_type}", count: 1),
+        title: @content_item.title,
+        average_title_length: "long" %>
+
+    <%= render 'shared/translations' %>
+    <% if @content_item.national_statistics? %>
+      <%= render 'shared/national_statistics_logo' %>
+    <% end %>
+
+    <%= render 'components/notice', @content_item.withdrawal_notice_component  %>
+    <%= render 'shared/metadata', content_item: @content_item %>
+    <%= render 'shared/history_notice', content_item: @content_item %>
+    <%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
+
+    <div class="grid-row">
+      <%= render 'components/heading', text: t("publication.documents", count: @content_item.documents_count), id: "documents-title" %>
+      <div aria-labelledby="documents-title">
+        <%= render 'govuk_component/govspeak',
+            content: @content_item.documents,
+            direction: page_text_direction %>
+      </div>
+    </div>
+
+    <div class="grid-row">
+      <%= render 'components/heading', text: t("publication.details"), id: "details-title" %>
+
+      <div aria-labelledby="details-title">
+        <%= render 'govuk_component/govspeak',
+            content: @content_item.details,
+            direction: page_text_direction %>
+      </div>
+    </div>
+  </div>
+
+  <%= render 'shared/sidebar_tasklist', tasklist: tasklist %>
+</div>
+
+
+<%= render 'shared/footer', @content_item.document_footer %>
```

Comment:
If the task list is fully expanded it pushes the footer to the bottom of the page, which is a bit odd...

![screen shot 2017-11-21 at 11 19 18](https://user-images.githubusercontent.com/861310/33069880-e03e71b4-cead-11e7-8f0a-523b9f6aaa3b.png)

It could be moved up inside the main grid element and placed at the bottom of the column-two-thirds element to fix this.

Reply:
@andysellick been addressed